flocker/provision/_install.py

```diff
@@ -793,7 +793,7 @@ def open_ufw(service):
         ])
 
 
-def task_enable_flocker_control(distribution):
+def task_enable_flocker_control(distribution, action="start"):
     """
     Enable flocker-control service.
```

Comment:
It's a bit unfair asking you to do this, since previous committers haven't but... could you please document the parameters to this function?